olp-cpp-sdk-core/include/olp/core/geo/tiling/TilingScheme.h

```diff
@@ -23,33 +23,28 @@
 
 namespace olp {
 namespace geo {
+
 template <class TSubdivisionScheme, class TProjection>
 class TilingScheme : public ITilingScheme {
  public:
   using SubdivisionScheme = TSubdivisionScheme;
   using Projection = TProjection;
 
-  TilingScheme() {}
+  TilingScheme() = default;
+  ~TilingScheme() override = default;
+
+  /// @copydoc ITilingScheme::GetSubdivisionScheme()
+  const ISubdivisionScheme& GetSubdivisionScheme() const override {
+    return sub_division_scheme_;
```

Comment:
```suggestion
    return subdivision_scheme_;
```
If `sub_division`, than the `ISubdivisionScheme` should be `ISubDivisionScheme`. 

Anyways, `subdivision` is the correct word